src/sec_related_rates.mbx

```diff
@@ -269,232 +269,303 @@
+            <p><var name="$a" width="10" /></p>
+            <!--FUNGUS: Are units required with the answer? (Yes) How do I force the user to enter them?-->
           </statement>
-          <solution>
-            <p><m>63.14</m>mph</p>
-          </solution>
         </webwork>
       </exercise>
+<!--Exercise 6: Also refer to Example 100-->
       <exercise>
         <webwork seed="1">
           <setup>
-            <var name="">
-              <static></static>
+            <var name="$a">
+              <static>64.44</static>
+            </var>
+            <var name="$b">
+              <static>78.89</static>
             </var>
             <pg-code>
+              $a = 64.44;
+              $b = 78.89;
             </pg-code>
           </setup>
           <statement>
             <p>Consider the traffic situation introduced in <xref ref="ex_rr3">Example</xref>. Calculate how fast the <q>other car</q> is traveling in each of the following situations.</p>
-
             <ol>
-              <li><p>The officer is traveling due north at 50mph and is 1/2 mile from the intersection, while the other car is 1 mile from the intersection traveling west and the radar reading is <m>-80</m>mph?</p></li>
-
-              <li><p>The officer is traveling due north at 50mph and is 1 mile from the intersection, while the other car is 1/2 mile from the intersection traveling west and the radar reading is <m>-80</m>mph?</p></li>
+              <li>
+                <p>The officer is traveling due north at 50mph and is 1/2 mile from the intersection, while the other car is 1 mile from the intersection traveling west and the radar reading is <m>-80</m>mph?</p>
+                <p><var name="$a" width="10"/></p>
+              </li>
+              <li>
+                <p>The officer is traveling due north at 50mph and is 1 mile from the intersection, while the other car is 1/2 mile from the intersection traveling west and the radar reading is <m>-80</m>mph?</p>
+                <p><var name="$b" width="10"/></p>
+              </li>
             </ol>
           </statement>
-          <solution>
-            <ol>
-              <li><p><m>64.44</m> mph</p></li>
-
-              <li><p><m>78.89</m> mph</p></li>
-            </ol>
-          </solution>
+<!--FUNGUS: There's no reason I need to keep the <solution> text, is there?-->
         </webwork>
       </exercise>
+<!--Exercise 7: Anti-aircraft gun-->
+<!--FUNGUS: Why does this exercise have xml:id? The others do not.-->
       <exercise xml:id="exer_04_02_ex_07">
         <webwork seed="1">
           <setup>
-            <var name="">
-              <static></static>
+<!--FUNGUS: Should I solve these to get the requisite 4 significant figures?-->
```

Comment:
Inside the setup tag, you no longer need `var` tags at all. (Or at least you won't in the very near future.)